src/corehost/cli/nethost/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+
+cmake_minimum_required (VERSION 2.6)
+project(nethost)
+
+set(DOTNET_PROJECT_NAME "nethost")
+
+# Include directories
+include_directories(../fxr)
+
+# CMake does not recommend using globbing since it messes with the freshness checks
+set(SOURCES
+    nethost.cpp
+    ../fxr_resolver.cpp
+    ../fxr/fx_ver.cpp
+)
+
+include(../lib.cmake)
+
+add_definitions(-DFEATURE_LIBHOST=1)
+add_definitions(-DNETHOST_EXPORT)
+
+install(TARGETS nethost DESTINATION corehost)
+install_symbols(nethost corehost)
```

Comment:
We should add an `install()` command to install the public `nethost.h` header for easy distribution. I think the `PUBLIC_HEADER` target property and install command is available in all the CMake versions we use in this repo.

Reply:
From what I read, it seems like `PUBLIC_HEADER` does different things on Apple. I made it put `nethost.h` right next to the .lib and .dll. we haven't dealt with the packaging/distribution side of things yet, but those three will need to go together, so I think at least putting them next to each other in bin makes sense for now.

Reply:
Sounds good!